### DISCUSSION TITLE
Deploy to github pages from pull requests

### DISCUSSION THREAD

**Discussion by ucalyptus2** [jekyll/jekyll#8747] [Q&A]
Basically https://github.com/travis-ci/dpl/issues/991 but want to do that jekyll websites on Gh pages such as https://github.com/explore-ml-iemk/explore-ml-iemk.github.io/pull/4

**Comment by datapolitical** [comment]
Use a GitHub action like https://github.com/jeffreytse/jekyll-deploy-action to build and deploy the branch
